### PR TITLE
[doc] Add code samples for Protobuf and Avro schemas in java documentation

### PR DESCRIPTION
**Pull request by vzhikserg**
### Motivation

We could not find Java examples which would include Avro and Protobuf schemas, but from the other [documentation page](https://pulsar.apache.org/docs/en/concepts-schema-registry/) it was clear that Pulsar supports them.

### Modification

Two code snippets (one for Protobuf and the other one for Avro) were added to the [Schema example](https://pulsar.apache.org/docs/en/client-libraries-java/#schemas) chapter.

### CODE REVIEW

File: site2/docs/client-libraries-java.md [hunk 1]
```diff
@@ -602,6 +603,24 @@ The following schema formats are currently available for Java:
         .create();
   ```
 
+* Protobuf schemas can be generate using the `ProtobufSchema` class. The following example shows how to create the Protobuf schema and use it to instantiate a new producer:
+
+  ```java
+  Schema<MyProtobuf> protobufSchema = ProtobufSchema.of(MyProtobuf.class);
+  Producer<MyProtobuf> protobufProducer = client.newProducer(protobufSchema)
```
Comment (merlimat) [inline site2/docs/client-libraries-java.md]: The preferred form should be: 

```java
 client.newProducer(Schema.PROTOBUF(MyProtobuf.class))
```

File: site2/docs/client-libraries-java.md [hunk 2]
```diff
@@ -602,6 +603,24 @@ The following schema formats are currently available for Java:
...
+        .topic("some-protobuf-topic")
+        .create();
+  ```
+
+* Avro schemas can be defined with the help of the `AvroSchema` class. The next code snippet demonstrates the creation and usage of the Avro schema:
+  
+  ```java
+  Schema<MyAvro> avroSchema = AvroSchema.of(MyAvro.class);
+  Producer<MyAvro> avroProducer = client.newProducer(avroSchema)
```
Comment (merlimat) [inline site2/docs/client-libraries-java.md]: Same here:

```java
client.newProducer(Schema.AVRO(MyProtobuf.class))
```

File: site2/docs/client-libraries-java.md (hunk 1 above)
Comment (vzhikserg) [reply site2/docs/client-libraries-java.md]: yes, I will change it. In my project the `Schema.PROTOBUF` form is used, but I wanted to keep it consistent with the sample for the JSON schema.

Comment (merlimat) [reply site2/docs/client-libraries-java.md]: Oh, I see. Do you mind also changing the example for JSON?

Comment (vzhikserg) [reply site2/docs/client-libraries-java.md]: sure, I will update it

**Pull request merged**